utils/obsproc/Ghrsst2Ioda.h

```diff
@@ -143,9 +145,13 @@ namespace gdasapp {
           // TODO(Somebody): add sampled std. dev. of sst to the total obs error
           obserror[i][j] = static_cast<float>(sstObsErr[index]) * errScaleFactor + errOffSet;
 
-          // epoch time in seconds
-          seconds[i][j]  = static_cast<float>(sstdTime[index]) * dtimeScaleFactor
-                         + static_cast<float>(refTime[0]);
+          // epoch time in seconds and avoid to use FillValue in calculation
+          if (sstdTime[index] == dtimeFillValue) {
+            seconds[i][j] = 0;
```

Comment:
how is storing 0 any better? Are these values not used when superobing?

Reply:
I would expect that a time with the fill value would indicate an observation to be filtered out

Reply:
Those are used and will be divided by counts,

https://github.com/NOAA-EMC/GDASApp/blob/401bd1e88c09934ada0a7f7184122ed280113d6d/utils/obsproc/superob.h#L54

At the end, those are filtered if less than zero
https://github.com/NOAA-EMC/GDASApp/blob/5a23b35ea9a2f0ed61a780c4cd2c08dbee876f44/utils/obsproc/Ghrsst2Ioda.h#L214

Am I thinking wrong?

Reply:
ha, ok. You mean those are NOT used.

Reply:
I'm still a bit paranoid @apchoiCMD , could you redo those histogram of time in the original and ioda files? Make sure we're not off again?

Reply:
> I would expect that a time with the fill value would indicate an observation to be filtered out

As @AndrewEichmann-NOAA is suggesting, why not filter out based on the fillvalue @apchoiCMD , as opposed to suing 0 ?

Reply:
> I'm still a bit paranoid @apchoiCMD , could you redo those histogram of time in the original and ioda files? Make sure we're not off again?

Expected and wanted to do, I am already on it-

Reply:
- 25 files of ABI_G16 in 20210702.00, Plot compares Raw and Ioda

![16_Before_After](https://github.com/NOAA-EMC/GDASApp/assets/141867620/6a5b2a19-428a-47ab-af56-b11a5059a868)

Reply:
Still does not look correct @apchoiCMD .